src/arch/amdgpu/vega/insts/vop3p_mai.cc

```diff
@@ -168,112 +168,6 @@ namespace VegaISA
             vdst[3][i] = result[(i/16)*4+3][(i%16)];
         }
 
-        for (int i = 0; i < 4; ++i) {
-            vdst[i].write();
-        }
-    } // execute
-    // --- Inst_VOP3P_MAI__V_MFMA_F64_16X16X4F64 class methods ---
-
-    Inst_VOP3P_MAI__V_MFMA_F64_16X16X4F64::
```

Comment:
@abmerop should we be moving vop3p_mai from this file to others like seems to be happening here?

Reply:
My opinion is we should make a new vop3p_mai.hh header with the template and aliases.  The instructions.hh already has 45k lines of code

Reply:
Right, this is exactly what I meant.